tests/smc/test_inner_kernel_tuning.py

```diff
@@ -14,7 +14,7 @@
 from blackjax import adaptive_tempered_smc, tempered_smc
 from blackjax.mcmc.random_walk import build_irmh
 from blackjax.smc import extend_params
-from blackjax.smc.inner_kernel_tuning import inner_kernel_tuning
+from blackjax.smc.inner_kernel_tuning import as_sampling_algorithm
```

Comment:
```suggestion
from blackjax.smc.inner_kernel_tuning import as_sampling_algorithm as inner_kernel_tuning
```

same for the rest of the test